tawkto/readme.txt

```diff
@@ -55,125 +55,129 @@ Note: You will need a free tawk.to account : [Create one for free here!](https:/
 
 == Changelog ==
 
-= 0.1.0 =
```

Comment:
Most of the changes here are only reorganizing the changelogs. Latest changelog comes first